egs/lre07/v2/lid

```diff
@@ -0,0 +1 @@
+/home/mousmita/WORK/LanguageID/kaldi/egs/lre07/v1/lid
```

Comment:
The symbolic link should be with respect to the example directory, rather than an absolute path in your homedir. E.g., something like ln -s ../v1/lid should do it.